### ISSUE TITLE
Gray hex value used instead of color variables

### ISSUE THREAD

**Issue by maya** [18F/web-design-standards#916]
We use a color `#757575` in a few instances for hint text and outlines of inputs (radio buttons and checkboxes). `#757575` is not a color variable we offer, but is the lightest gray that can be used which passes color contrast requirements. The closest variable would be `$color-gray` which is slightly darker.

Should we use one of our color variables or add a new variable for this gray?

Here is what they look like side-by-side:

<img width="340" alt="screen shot 2015-12-07 at 4 27 17 pm" src="https://cloud.githubusercontent.com/assets/5249443/11644333/e4037b96-9cff-11e5-9ba5-23e68ec59801.png">

**Comment by mollieru** [comment]
These shades are different enough that I'm wondering if we should add the #757575 to the color palette.  Rationale: looking at the two grays in context, #757575 is light enough that when next to our black, it recedes to support a bit more distinguishable visual hierarchy.  See for example the difference below:

![image](https://cloud.githubusercontent.com/assets/3680494/11787889/1fa2a7b0-a25c-11e5-96bb-bf324ce8431a.png) 
![image](https://cloud.githubusercontent.com/assets/3680494/11787896/25fae014-a25c-11e5-97be-0650edf6f952.png)

But I'm curious what other folks think about this -- @jenniferthibault @jehlers @rtwell @jjoteal ?

**Comment by mollieru** [comment]
If so, thinking we might name is `gray-medium` -- thoughts/ideas on this welcome, too.

**Comment by jenniferthibault** [comment]
I'd be curious to know why you then need the darker grey. Could #757575 replace #5b616b ? I don't know your use cases for #5b616b , but I see the benefit of #757575 with black in the screenshots above.

**Comment by Drakenhart** [comment]
Would it be for designer flexibility@mollieru?
This project has a nice, wide range of color choices. I could see it being made and added color just for designer choice.

**Comment by jenniferthibault** [comment]
An alternative! If you do need multiple greys....

Your existing grey #5b616b is a cool blue grey, while #757575 is a grey with much more red in it, which is why they feel like they are mismatched when put next to each other. To keep it more in-palette, try a lighter blue-based grey to match your ##5b616b.

Here's what I mean.
Current:
![screen shot 2015-12-14 at 12 40 20 pm](https://cloud.githubusercontent.com/assets/11636908/11788597/e376b688-a25f-11e5-86e2-a3b41ae30df4.png)

<img width="215" alt="screen shot 2015-12-14 at 12 42 43 pm" src="https://cloud.githubusercontent.com/assets/11636908/11788685/4fab0f20-a260-11e5-9997-3415c94fb7b0.png">


With grey #656970, in the blue family. With white, color contrast ratio of 5.51:1 :

![screen shot 2015-12-14 at 12 41 56 pm](https://cloud.githubusercontent.com/assets/11636908/11788654/210aae64-a260-11e5-8e20-451a75317e9d.png)

<img width="206" alt="screen shot 2015-12-14 at 12 42 49 pm" src="https://cloud.githubusercontent.com/assets/11636908/11788680/4ad591b4-a260-11e5-8b53-02828f54b134.png">


You may find that you need it lighter still, so play around, but I'd recommend exploring within this principle.

**Comment by jehlers** [comment]
@jenniferthibault's explorations are awesome. 

I've used #767676 before which is basically the same as #757575. I think the red-er shade is helpful to contrast more with the black in cases like your hint text above. But I could see more than one gray palate that a designer could choose from depending on if their brand is on the warmer or colder color side of the spectrum.

**Comment by maya** [comment]
It's a very subtle difference...

#767676
<img width="258" alt="screen shot 2015-12-14 at 10 54 25 am" src="https://cloud.githubusercontent.com/assets/5249443/11790527/2bb3c0c6-a251-11e5-9cae-588bee3d907a.png">

#757575
<img width="238" alt="screen_shot_2015-12-08_at_3 07 27_pm" src="https://cloud.githubusercontent.com/assets/5249443/11790587/6d8a938a-a251-11e5-95ff-bdeae06cf140.png">

cc: @jenniferthibault @jehlers 

Happy to go either way you think is best!

**Comment by maya** [comment]
@jenniferthibault `#757575` has equal amounts of RGB making it a neutral gray. It appears more red next to the bluer gray, but I think that's just how the human eye perceives color.

**Comment by maya** [comment]
Talked to @jehlers and @angel who both liked `#757575` bc it provides higher contrast against the black.

**Comment by jpyuda** [comment]
I think we should consider `#787878`

**Comment by maya** [comment]
We went with `#757575` for now in #917, but if you find a better shade, let us know and we can change.

**Comment by maya** [comment]
Closed via #917.
